sycl/include/sycl/backend/opencl.hpp

```diff
@@ -25,6 +27,11 @@ __SYCL_EXPORT context make_context(pi_native_handle NativeHandle);
 __SYCL_EXPORT queue make_queue(const context &Context,
                                pi_native_handle InteropHandle);
 
+__SYCL_EXPORT bool has_extension(const sycl::platform &syclPlatform,
+                                 const std::string &extension);
+__SYCL_EXPORT bool has_extension(const sycl::device &syclDevice,
+                                 const std::string &extension);
```

Comment:
```suggestion
__SYCL_EXPORT bool has_extension(const sycl::platform &SyclPlatform,
                                 const std::string &Extension);
__SYCL_EXPORT bool has_extension(const sycl::device &SyclDevice,
                                 const std::string &Extension);
```
Minor.

Reply:
I have now resolved this in commit [b2e2c43](https://github.com/intel/llvm/pull/7059/commits/b2e2c43210194e3534ca24121395c4d6adf2efe3) that I have just rebased.